forest/drivers/intake_loader.py

```diff
@@ -18,12 +18,29 @@
 except ModuleNotFoundError:
     intake = None
 
+import forest.view
 from forest import geo
-from forest.drivers import gridded_forecast
+from forest.drivers.gridded_forecast import _to_datetime
```

Comment:
This line will have to change to `import forest.util.to_datetime as _to_datetime` I think